ourhexenv.py

```diff
@@ -183,43 +233,43 @@ def render(self, mode="human"):
-            for col in range(self.board_size):
-                if self.board[0, col] == marker:
-                    if self.dfs(marker, 0, col, visited):
-                        return True
-        else:
-            # Check DFS from all top row cells to see if Player 2 has reached the bottom
-            for row in range(self.board_size):
-                if self.board[row, 0] == marker:
-                    if self.dfs(marker, row, 0, visited):
-                        return True
-        return False
-
-    def dfs(self, marker, row, col, visited):
-        if marker == 1 and row == self.board_size - 1:  # Player 1 reached bottom
-            return True
-        if marker == 2 and col == self.board_size - 1:  # Player 2 reached right side
-            return True
-
-        directions = [(-1, 0), (1, 0), (0, -1), (0, 1), (-1, 1), (1, -1)]
-        visited.add((row, col))
-
-        for dr, dc in directions:
-            new_row, new_col = row + dr, col + dc
-            if (
-                0 <= new_row < self.board_size
-                and 0 <= new_col < self.board_size
-                and (new_row, new_col) not in visited
-                and self.board[new_row, new_col] == marker
-            ):
-                if self.dfs(marker, new_row, new_col, visited):
-                    return True
-
-        return False
+    # def check_winner(self, marker):
+    #     visited = set()
+    #     if marker == 1:
+    #         # Check DFS from all top row cells to see if Player 1 has reached the bottom
+    #         for col in range(self.board_size):
+    #             if self.board[0, col] == marker:
+    #                 if self.dfs(marker, 0, col, visited):
+    #                     return True
+    #     else:
+    #         # Check DFS from all top row cells to see if Player 2 has reached the bottom
+    #         for row in range(self.board_size):
+    #             if self.board[row, 0] == marker:
+    #                 if self.dfs(marker, row, 0, visited):
+    #                     return True
+    #     return False
+    #
+    # def dfs(self, marker, row, col, visited):
+    #     if marker == 1 and row == self.board_size - 1:  # Player 1 reached bottom
+    #         return True
+    #     if marker == 2 and col == self.board_size - 1:  # Player 2 reached right side
+    #         return True
+    #
+    #     directions = [(-1, 0), (1, 0), (0, -1), (0, 1), (-1, 1), (1, -1)]
+    #     visited.add((row, col))
+    #
+    #     for dr, dc in directions:
+    #         new_row, new_col = row + dr, col + dc
+    #         if (
+    #             0 <= new_row < self.board_size
+    #             and 0 <= new_col < self.board_size
+    #             and (new_row, new_col) not in visited
+    #             and self.board[new_row, new_col] == marker
+    #         ):
+    #             if self.dfs(marker, new_row, new_col, visited):
+    #                 return True
+    #
+    #     return False
```

Comment:
Dead code, consider removing it instead of commenting out. If we want to bring back this code, we can do a revert on the hash and deal with the (possible) merge conflict